### ISSUE TITLE
load_object: Symbol "nn_tilde_setup" not found

### ISSUE THREAD

**Issue by infolithium** [acids-ircam/nn_tilde#53]
On Windows 10 with Pure Data Vanilla "load_object: Symbol "nn_tilde_setup" not found" shows up and the nn~ object can't be created. I followed the install directions in the readme. Am I doing something wrong?

**Comment by infolithium** [comment]
Nothing changed. on ver 1.5.5

**Comment by caillonantoine** [comment]
Have you tried compiling nn~ yourself ?

**Comment by infolithium** [comment]
`cmake ..\src -A x64 -DCMAKES_PREFIX_PATH="C:\libtorch" -DPUREDATA_BIN_DIR="C:\Program Files\Pd\src" -DPUREDATA_BIN_DIR="C:\Program Files\Pd\bin"`

`-- Selecting Windows SDK version 10.0.22000.0 to target Windows 10.0.19044.
CMake Error at backend/CMakeLists.txt:4 (find_package):
  By not providing "FindTorch.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Torch", but
  CMake did not find one.

  Could not find a package configuration file provided by "Torch" with any of
  the following names:

    TorchConfig.cmake
    torch-config.cmake

  Add the installation prefix of "Torch" to CMAKE_PREFIX_PATH or set
  "Torch_DIR" to a directory containing one of the above files.  If "Torch"
  provides a separate development package or SDK, be sure it has been
  installed.


-- Configuring incomplete, errors occurred!`

I've barely used CMake and never done anything with Libtorch so I'm not entirely sure what all the terminology here means. Adding "C:\libtorch\cmake\Torch" to the CMAKE_PREFIX_PATH environment didn't change anything

**Comment by caillonantoine** [comment]
Download [libtorch](https://download.pytorch.org/libtorch/cpu/libtorch-win-shared-with-deps-2.0.1%2Bcpu.zip) and unzip it somewhere on your computer (but maybe not directly in `c:`), somewhere in the downloads folder should be fine. Then you can build the project using

```bash
cmake ..\src -A x64 -DCMAKE_PREFIX_PATH="C:\Users\XXX\Downloads\libtorch"
cmake --build . --config Release
```

**Comment by infolithium** [comment]
The location of libtorch didn't have anything to do with it, I also downloaded the right .zip specified in the readme. The real problem is that one of the flags was misspelled and I didn't notice. I wasn't paying attention, sorry about that. additionally I typed DPUREDATA_BIN_DIR twice instead of PUREDATA_INCLUDE_DIR, and because of that I would've only built files for Max/MSP. 

Building from source seemed to have solved the issue
